service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -1124,45 +1125,76 @@ public Map<String, Object> clearCloseCaseTabBackupField(CaseData caseData) {
         return Map.of("closeCaseTabField", closeCaseField.toBuilder().dateBackup(null).build());
     }
 
-    public Map<String, Object> migrateHearingType(CaseData caseData) {
-        if (nonNull(caseData.getHearingDetails())) {
-            List<Element<HearingBooking>> hearingDetails = caseData.getHearingDetails();
-            for (Element<HearingBooking> hearings: hearingDetails) {
-                HearingBooking hearingBooking = hearings.getValue();
-                if (OTHER.equals(hearingBooking.getType())) {
-                    Optional<HearingType> hearingType = evaluateType(hearingBooking.getTypeDetails());
-                    if (hearingType.isPresent()) {
-                        hearingBooking.setType(hearingType.get());
-                    } else {
-                        hearingBooking.setType(FURTHER_CASE_MANAGEMENT); // default to further case management
-                    }
-                }
+    private static void processHearingBooking(Element<HearingBooking> element) {
+        HearingBooking hearingBooking = element.getValue();
+        if (OTHER.equals(element.getValue().getType())) {
+            Optional<HearingType> hearingType = evaluateType(hearingBooking.getTypeDetails());
+            if (hearingType.isPresent()) {
+                hearingBooking.setType(hearingType.get());
+            } else {
+                hearingBooking.setType(FURTHER_CASE_MANAGEMENT);
             }
```

Comment:
can probably change this to something like 
```suggestion
            hearingBooking.setType(hearingType.orElse(FURTHER_CASE_MANAGEMENT));
```

to replace the if/else! (some people dislike that pattern though so personal taste 😆)